tests/pipelines/test_pipelines_video_classification.py

```diff
@@ -39,6 +40,13 @@
 class VideoClassificationPipelineTests(unittest.TestCase):
     model_mapping = MODEL_FOR_VIDEO_CLASSIFICATION_MAPPING
 
+    if _run_pipeline_tests:
+        # we use revision="refs/pr/1" until the PR is merged
+        # https://hf.co/datasets/hf-internal-testing/fixtures_image_utils/discussions/1
```

Comment:
```suggestion
```
(I think this is copy/paste from the previous related PR :) )

Reply:
ah yes ... nice catch I will remove this , push then merge directly. Don't want to waste CircleCI resouce again :-)